xc/device.mli

```diff
@@ -80,7 +80,8 @@ sig
 	       -> mac:string -> carrier:bool 
 	       -> ?mtu:int -> ?rate:(int64 * int64) option
 	       -> ?protocol:protocol -> ?backend_domid:Xenctrl.domid 
-	       -> ?other_config:((string * string) list) 
+	       -> ?other_config:((string * string) list)
+	       -> ?static_ip_setting:((string * string) list)
```

Comment:
If `/local/domain/<domid>/xenserver/` is going to be a new path for all kinds of config, beyond IP setting, then we should make the device.ml(i) code generic as well (as done for `extra_private_keys`). We should not have anything specific, such as `static_ip_setting` in here, but have that in xenops_server_xen.ml.